### PR TITLE
Doxygen in CALRAD_WCLOUD_newcrtm.f CALRCH.f CALSTRM.f CALTAU.f CALTHTE.f CALUPDHEL.f.

### PR DESCRIPTION
**Pull request by kayeekayee**
This is part of Issue #392.

Fixes the doxygen warnings in CALRAD_WCLOUD_newcrtm.f CALRCH.f
CALSTRM.f CALTAU.f CALTHTE.f CALUPDHEL.f.

Please review this subroutine. Thank you.

### CODE REVIEW

**Review by edwardhartnett — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
@kayeekayee PR #466 was just committed. Can you sync your branch update to date for merging. Thanks!

File: sorc/ncep_post.fd/CALUPDHEL.f [hunk 1]
```diff
@@ -1,39 +1,18 @@
 !> @file
-!
```
Comment (WenMeng-NOAA) [inline sorc/ncep_post.fd/CALUPDHEL.f:2]: @kayeekayee It seems to me your branch revert the changes in PR #458. Can you modify?

File: sorc/ncep_post.fd/CALRAD_WCLOUD_newcrtm.f [hunk 2]
```diff
@@ -1,30 +1,22 @@
...
+!>     
+!> ### Program history log:
+!> Date | Programmer | Comments
+!> -----|------------|---------
+!> 2007-01-17 | H Chuang       | Initial
+!> 2011-02-06 | Jun Wang       | add grib2 option
+!> 2014-12-09 | WM Lewis       | added function EFFR to compute effective particle radii channel selection using LVLS from WRF_CNTRL.PARM
+!> 2019-04-01 | Sharon Nebuda  | Added output option for GOES-16 & GOES-17 ABI IR Channels 7-16
+!> 2020-04-09 | Tracy Hertneky | Added Himawari-8 AHI CH7-CH16
+!> 2021-01-10 | Web Meng       | Added checking points for skiping grids with filling value spval
```
Comment (WenMeng-NOAA) [inline sorc/ncep_post.fd/CALRAD_WCLOUD_newcrtm.f]: @kayeekayee Can you correct the tybo "Web Meng" to "Wen Meng"here?

**Comment by kayeekayee** [comment]
Thank you so much for reviewing and catching all the problems, @WenMeng-NOAA.

**Review by WenMeng-NOAA — APPROVED** [review]

**Comment by WenMeng-NOAA** [comment]
The UPP RT tests were done on Hera. No base line update is needed in this PR.
